src/main/resources/net/praqma/jenkins/configrotator/scm/git/GitTarget/config.jelly

```diff
@@ -1,29 +1,19 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
-    <tr>
-        <td>
-            Name <f:textbox field="name"/>
-        </td>
-        <td>
-            Branch
-            <f:textbox field="branch"/>
-        </td>
-        <td>
-            Repository
-            <f:textbox field="repository"/>
-        </td>
-        <td>
-            Commit
-            <f:textbox field="commitId"/>
-        </td>
-        <td>
-            <tr>
-                <td>
-                    Fixed
-                    <f:checkbox field="fixed"/>
-                </td>
-            </tr>
-        </td>
-    </tr>
+    <f:entry title="Name">
+        Name <f:textbox field="name"/>
```

Comment:
is name intentionally duplicated here off of the title?
```suggestion
        <f:textbox field="name"/>
```

Reply:
🤦‍♂️, that's me just being up way too late trying to be productive.